docs/decisions/0018-standarize-django-po-files.rst

```diff
@@ -0,0 +1,157 @@
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+#. The edx-platform splits its translations files to ``mako-studio.po``,
+   ``django-partial.po`` among 11 files in total to make it possible for
+   translators to focus on learner-facing text
+   and de-prioritize educator-facing text. The idea was that LMS would be
+   translated into more languages than CMS (Studio) would.
+
+#. The edx-platform uses the ``i18n_tools segment`` to split the ``django.po``
+   and ``djangojs.po`` files into smaller files. This is done with the goal of
+   making it easier for translators to translate the files.
+
+#. When pulling from Transifex or cutting a new release, the translations are
+   bundled into 2 files (``django.po`` and ``djangojs.po``).
+
+#. The `FC-0012 project <https://openedx.atlassian.net/l/cp/XGS0iCcQ>`_
+   which implements `Translation Infrastructure update OEP-58 <https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification>`_
+   is in progress which means that all translation files are going to live in
+   the new `openedx-translations Transifex project`_.
+
+#. The `openedx-translations GitHub repository`_
+   is going to sync the translations between repos and Transifex as well as
+   store the translation files in centralized location as accepted in
+   `Translation Infrastructure update OEP-58 <https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification>`_.
+
+#. Several special steps are needed to for the `extract-translation-source-files.yml`_
+   GitHub workflow in the `openedx-translations GitHub repository`_
+   to extract the files from the edx-platform repo.
+   This seems redundant and and it should be possible to simplify the process.
+
+Decision: Only use two files for edx-platform translations (``django.po`` and ``djangojs.po``)
+----------------------------------------------------------------------------------------------
+
+edX Platform will push only two files (``django.po`` and
+``djangojs.``) to the `openedx-translations Transifex project`_.
+
+Consequences
+------------
+
+Pros:
+
+- Translators will need to locate only two resources for the ``edx-platform``
+  repo as opposed to 11 resources.
+- Simplify the ``edx-platform`` translation extraction scripts in the
+  `extract-translation-source-files.yml`_ GitHub workflow.
+- The release cut process will be simplified as well by removing the
+  translation merger step altogether.
+- Translators will need one workflow for both latest (master) and named
+  releases for the ``edx-platform`` repo.
+- In this decision we argue that the cost of splitting the resources outweighs
+  the benefits and propose to reduce the number of Transifex resources in the
+  ``openedx-translations`` Transifex project, which simplifies the work of both
+  Build Test Release and Transifex working groups.
```

Comment:
I'm not sure if this fits in this section. It feels more like a summary than a listed "Pro". That being said, we could probably just shorten it to

```suggestion
- Simplifies the work of both the Build Test Release and Transifex working groups.
```